src/main/java/Testcases/Railway/GeneralTest.java

```diff
@@ -7,6 +7,7 @@
 import org.testng.annotations.BeforeMethod;
 
 public class GeneralTest {
+    //comment
```

Comment:
Comment can be used as a description of the next row.
This one does not show any value, please remove